### ISSUE TITLE
Create nightly builds

### ISSUE THREAD

**Issue by alahj** [kimitzu/kimitzu-client#44]
**Is your feature request related to a problem? Please describe.**
In order to speed up the test/release cycle, please set the build scripts to build a new release every day.   Automatically update website with link to nightly builds.

**Comment by 24thsaint** [comment]
![image](https://user-images.githubusercontent.com/8315544/72238019-7a2ae280-3617-11ea-8348-d3d456857ee3.png)

Nightly builds are feasible, but auto-builds for MAC require payment on CircleCI. 

We may be able to build it manually on MAC by the end of the day, this means, MAC builds will be delayed compared to win/linux.

Shall we continue?

**Comment by alahj** [comment]
Yes, go ahead to build nightly builds for Window and Linux.  The Mac builds can be delayed if necessary for now.

**Comment by 24thsaint** [comment]
Reopening for MAC automated build implementation.
